config/superores/resources/assets/superores/lang/ja_jp.json

```
@@ -0,0 +1,31 @@
+{
+    "block.superores.super_coal_ore":"高密度石炭鉱石",
+    "block.superores.super_deepslate_coal_ore":"深層高密度石炭鉱石",
+    "block.superores.super_copper_ore":"高密度銅鉱石",
+    "block.superores.super_deepslate_copper_ore":"深層高密度銅鉱石",
+    "block.superores.super_diamond_ore":"高密度ダイアモンド鉱石",
+    "block.superores.super_deepslate_diamond_ore":"深層高密度ダイアモンド鉱石",
+    "block.superores.super_emerald_ore":"高密度エメラルド鉱石",
+    "block.superores.super_deepslate_emerald_ore":"深層高密度エメラルド鉱石",
+    "block.superores.super_fluorite_ore":"高密度蛍石鉱石",
+    "block.superores.super_deepslate_fluorite_ore":"深層高密度蛍石鉱石",
+    "block.superores.super_gold_ore":"高密度金鉱石",
+    "block.superores.super_deepslate_gold_ore":"深層高密度金鉱石",
+    "block.superores.super_iridium_ore":"高密度イリジウム鉱石",
+    "block.superores.super_iron_ore":"高密度鉄鉱石",
+    "block.superores.super_deepslate_iron_ore":"深層高密度鉄鉱石",
+    "block.superores.super_lapis_lazuli_ore":"高密度ラピスラズリ鉱石",
+    "block.superores.super_deepslate_lapis_lazuli_ore":"深層高密度ラピスラズリ鉱石",
+    "block.superores.super_lead_ore":"高密度鉛鉱石",
+    "block.superores.super_deepslate_lead_ore":"深層高密度鉛鉱石",
+    "block.superores.super_nether_gold_ore":"高密度ネザー金鉱石",
+    "block.superores.super_nether_quartz_ore":"高密度ネザークォーツ鉱石",
+    "block.superores.super_osmium_ore":"高密度オスミウム鉱石",
+    "block.superores.super_deepslate_osmium_ore":"深層高密度オスミウム鉱石",
+    "block.superores.super_redstone_ore":"高密度レッドストーン鉱石",
+    "block.superores.super_deepslate_redstone_ore":"深層高密度レッドストーン鉱石",
+    "block.superores.super_tin_ore":"高密度スズ鉱石",
+    "block.superores.super_deepslate_tin_ore":"深層高密度スズ鉱石",
```

[Mekanismの翻訳](https://github.com/mekanism/Mekanism/blob/1.20.x/src/main/resources/assets/mekanism/lang/ja_jp.json#L373)に合わせます
```suggestion
    "block.superores.super_tin_ore":"高密度錫鉱石",
    "block.superores.super_deepslate_tin_ore":"深層高密度錫鉱石",
```